app/routes/index.tsx

```diff
@@ -21,7 +21,8 @@ import {reloadInBackgroundIfNeeded} from '~/server-utils/kv-cache'
 export const loader = async ({request}: Parameters<LoaderFunction>[0]) => {
   try {
     const initialQuestionsData = await loadInitialQuestions(request)
-    return {initialQuestionsData}
+    const glossaryData = await loadGlossary(request)
```

Comment:
Hm, I don't like fetching all answers from Coda before sending the first byte to the client 🤔 

Can you reuse `onSiteAnswersRef` in a way similar to how `<Search ...` component used the same lazy-loaded data?

Reply:
I initially tried that way. The problem is with the contents, which don't get loaded until a given question is fetched. So this would require either doing some magic to load questions when someone hovers over a definition, or prefetching the questions that are used for the glossary. Coda's filtering options are rudimentary, to say the best, so the best alternative I could think of would be to first fetch the glossary tag, then get all questions that it uses, which would quickly blow up :/
My assumption here is that the glossary will be cached after the first invocation, so should be fast after that

Reply:
I still don't like making multiple requests from CF worker "backend" to Coda that fetch all data from all answers table, 1 should be enough and we should share the data for all features that need all data ... but I guess I can investigate duplicate request after (or ehm, by..) fixing the cache issues.